sdk/src/rest/commands/update/dashboards.ts

```diff
@@ -34,6 +34,29 @@ export const updateDashboards =
 		};
 	};
 
+/**
+ * Update multiple dashboards as batch.
+ * @param items
+ * @param query
+ * @returns Returns the dashboard objects for the updated dashboards.
+ * @throws Will throw if no items are provided
+ */
+export const updateDashboardsBatch =
+	<Schema, const TQuery extends Query<Schema, DirectusDashboard<Schema>>>(
+		items: Partial<DirectusDashboard<Schema>>[],
```

Comment:
I think we need to use `NestedPartial` for most of the update batch functions, as nested objects can be, for example
- an update (only containing primary key + fields to update)
  ```js
  updateDashboardsBatch([
    {
      id: "0fed4f30-b500-4457-9c71-e00797ce973b",
      user_created: {
        id: "bb9342a4-9454-4cb5-8914-c80d720e4e02",
        first_name: "updated via update batch",
      },
    },
  ])
  ```
- a new item creation (not containing primary key + probably omitting other fields)
  ```js
  updateDashboardsBatch([
    {
      id: "0fed4f30-b500-4457-9c71-e00797ce973b",
      user_created: {
        first_name: "created via update batch",
      },
    },
  ])
  ```

```suggestion
		items: NestedPartial<DirectusDashboard<Schema>>[],
```